src/styles/NavBar.js

```diff
@@ -0,0 +1,72 @@
+import { NavLink as Link } from 'react-router-dom';
+import styled from 'styled-components';
+  
+export const Nav = styled.nav`
```

Comment:
Bice, we also use styled components this should have gave you an idea on how we use them.